main.nf

```diff
@@ -924,7 +926,7 @@ process featureCounts {
     // Try to get real sample name
     sample_name = bam_featurecounts.baseName - 'Aligned.sortedByCoord.out'
     """
-    featureCounts -a $gtf -g gene_id -o ${bam_featurecounts.baseName}_gene.featureCounts.txt -p -s $featureCounts_direction $bam_featurecounts
+    featureCounts -a $gtf -g gene_id -o ${bam_featurecounts.baseName}_gene.featureCounts.txt --extraAttributes ${params.fcExtraAttributes} -p -s $featureCounts_direction $bam_featurecounts
```

Comment:
Currently it's impossible _not_ to specify this. Maybe nicer to make it an optional flag? eg:

```nextflow
def avail_mem = task.memory ? "--limitGenomeGenerateRAM ${task.memory.toBytes() - 100000000}" : ''
def extraAttributes = params.fcExtraAttributes ? "--extraAttributes ${params.fcExtraAttributes}" : ''
"""
featureCounts -a $gtf -g gene_id -o ${bam_featurecounts.baseName}_gene.featureCounts.txt $extraAttributes
```

Also, it would be nice to split this command into a multi-line for better readability perhaps.

Reply:
Done!